frontend/packages/dev-console/integration-tests/features/Helm/default-HelmCharts-Installation.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Helm Chart
+    User will be able to install the helm charts
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: The Helm Chart option on the +Add Page: HR-01-TC01
+    Given user is at +Add page
+    Then user can see Helm Chart card on the +Add page
+
+
+@regression, @smoke, @ODC-3022, @ODC-2993
+Scenario: Install Helm Chart from +Add Page using Form View: HR-02-TC01
+    Given user is at +Add page
+    When user clicks on the Helm Chart card on the +Add page
+    And user searches for the 'Node-ex-k' helm chart 
+    And user clicks on the 'Node-ex-k' helm chart card
+    And user clicks on the Install Helm Chart button
+    And user selects the Form View
+    And user enters the required field
+    And user updates the Chart Version
+    And user clicks on the Install button
+    Then user is redirected to Topology page
+    And Topology page have the helm chart workload 
+
+
+@regression, @ODC-3022, @ODC-2993
+Scenario: Install Helm Chart from Developer Catalog Page using YAML View: HR-03
```

Comment:
Update with test case number, for consistency purpose
If you have only one test case, still I recommend you to add test case number like HR-03-TC01

Reply:
Updated the test case number